eng-paths/sr_software_engineer_3.md

```diff
@@ -0,0 +1,381 @@
+- Mentors engineers and fosters their growth.
+- Coaches teams to continuously improve delivery pace and quality.
+- Can step in as a contributor on any team when needed.
+- Able to suggest when/how projects could take shortcuts and still maintain quality to satisfy our customers.
+- Self-guided in a professional way, do their own prioritization when needed.
+- Uses data and facts to debate technical architecture with team members and stakeholders to gain buy-in.
+- Take the lead on following up on the incidents that happened, proactively looking into ways to improve the systems to avoid those same issues from happening again, and/or improve monitoring to detect those sooner whenever applicable.
+- Fosters cost-awareness by advocating for initiatives that lead to savings, and diligently monitoring spending across existing systems.
+- Comfortable with navigating ambiguity, being able to define the next steps of own scope that align with Engineering and the company's needs.
+
+</td></tr>
+</table>
+
+
+### Communication and leadership
+<table  markdown="1">
+<tr><td>
+
+Sr Software Engineer 2
+
+</td></tr>
+<tr><td  markdown="1">
+
+- Makes others better through code reviews, thorough documentation, technical guidance, and mentoring or serving as a Tech Lead on a project. .
+- Understands the tradeoffs between technical and product needs to lead to solutions that take all of these needs into account and drive business outcomes.
+- Identifies and proposes strategies around technical problems affecting their team, communicates standards, and gets buy-in on solutions.
+- Able to discuss complex technical issues at an abstract level with non-technical people.
+- Strongly influences and leads team culture.
+- During critical outages and high-impact issues, clearly communicate the status and high-level impact in the appropriate public channels for visibility and awareness.
+- After a critical outage or high-impact issue is resolved, help debrief the outcomes through messages on the proper channels and/or post-mortem documents and create follow-up action items (when applicable) as a measure to prevent the issue from happening again or to be able to detect it sooner.
+- Able to analyze and execute cross-team engineering initiatives by leading the scope of implementation within own team.
+- Communicates clearly with all stakeholders, being able to articulate trade-offs and shift the priorities when necessary.
+- Helps to create buy-in on the decisions (together with the EM and PM) by helping the team to understand the "why" of the decisions being taken.
+
+</td></tr>
+<tr><td style="font-weight:bold">
+
+Sr Software Engineer 3
+
+</td></tr>
+<tr><td markdown="1" style="font-weight:bold">
+
+- Thrives in communication, ensuring clarity and objectiveness for individuals with varying levels of technical expertise.
+- Able to clearly articulate pros/cons, costs and tradeoffs of possible decisions.
```

Comment:
What about `Able to clearly articulate pros and cons of possible options, including: costs, tradeoffs, risks, and benefits`?